fem/plinearform.cpp

```diff
@@ -14,6 +14,7 @@
 #ifdef MFEM_USE_MPI
 
 #include "fem.hpp"
+using namespace std;
```

Comment:
Without any other changes in this file, this addition seems unnecessary.

Reply:
Agreed.  I don't recall how that came about.  I'll take it out.